addons/sale_pdf_quote_builder/models/sale_order.py

```diff
@@ -19,16 +19,19 @@ def _default_quotation_document_ids(self):
         string="Available Quotation Documents",
         comodel_name='quotation.document',
         compute='_compute_available_quotation_document_ids',
+        groups='sales_team.group_sale_salesman',
     )
     is_pdf_quote_builder_available = fields.Boolean(
         compute='_compute_is_pdf_quote_builder_available',
+        groups='sales_team.group_sale_salesman',
```

Comment:
Same here, knowing there might be some documents attached should be available if you can see the SO